City-of-Helsinki/drupal-helfi-platform-config
# UHF-2478: updated genesys chat

How to test:

- Setup a local environment
- Run this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-2478_updated_genesys_chat`
- Place the "Genesys Chat" block in the "Attachments" region (on all pages by default) in block config & set "Display title" to disabled
- Go to any content page and see the chat appear in the lower right corner

NOTE: The chat runs, but doesn't seem to function correctly at least in local environments. Needs to be deployed to dev to see how it works there.
